migrations/frontend/1669297489_add_permission_sync_jobs/up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS permission_sync_jobs (
+  id                SERIAL PRIMARY KEY,
+  state             text DEFAULT 'queued',
```

Comment:
should we make it not null?
```suggestion
  state             text not null DEFAULT 'queued',
```

Reply:
I don't think it matters that much with the default value. I followed the docs: https://docs.sourcegraph.com/dev/background-information/workers That's how I ended up with this schema.